draft-ietf-ppm-dap.md

```diff
@@ -1404,7 +1404,7 @@ enum {
   task_expired(7),
   invalid_message(8),
   report_too_early(9),
-  (255)
+  (65535)
```

Comment:
Can you explain the choice to widen this error type? Are we ever going to define this many distinct error codes? Admittedly errors should be the minority of messages so this extra byte won't cost extra bandwidth in the common case of the protocol.